Sources/FoundationEssentials/FileManager/FileManager+Basics.swift

```diff
@@ -16,6 +16,28 @@ import Darwin
 import Glibc
 #elseif os(Windows)
 import CRT
+import WinSDK
+#endif
+
+#if os(Windows)
+extension _FILE_ID_128 /* : @retroactive Equatable */ {
+    internal static func == (_ lhs: _FILE_ID_128, _ rhs: _FILE_ID_128) -> Bool {
```

Comment:
Will this cause any ambiguity if in the future something from below Foundation adds `_FILE_ID_128.==` as part of an equatable conformance? Just want to double check that we don't need to name this something like `_equals` just so that we don't conflict in that probable future

Reply:
Sure, I can rename this to `_equals`.